### PR TITLE
C353529 Verify progress bar for uploading identifier files(firebird) …

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C353529](https://foliotest.testrail.io/index.php?/cases/view/353529)
[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5439/allure)
image

### CODE REVIEW

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-progress-bar.cy.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      'C353529 Verify progress bar for uploading identifier files(firebird) (TaaS)',
+      { tags: [testTypes.smoke, devTeams.firebird] },
+      () => {
+        BulkEditSearchPane.checkUsersRadio();
+        BulkEditSearchPane.selectRecordIdentifier('User Barcodes');
+
+        BulkEditSearchPane.verifyDragNDropUsersBarcodesArea();
+        BulkEditSearchPane.uploadFile(userBarcodesFileName);
+        BulkEditSearchPane.waitFileUploading();
+        BulkEditSearchPane.verifyPopulatedPreviewPage();
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-progress-bar.cy.js:52]: please verify the progress bar described in the second step of the TC

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 2]
```diff
@@ -80,7 +80,11 @@ export default {
   actionsIsAbsent() {
     cy.expect(actions.absent());
   },
-
+  verifyPopulatedPreviewPage() {
```
Comment (manowarqqq) [inline cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:83]: Please, use array of assertions
Example:
 cy.expect([
      deleteUploadedFileModal.exists(),
      deleteUploadedFileModal.find(HTML(including(modalContent))).exists(),
      deleteUploadedFileModal.find(modalNoButton, { disabled: true }).exists(),
      deleteUploadedFileModal.find(modalYesButton, { disabled: false }).exists(),
    ]);

**Review by manowarqqq — APPROVED** [review]

**Review by manowarqqq — COMMENTED** [review]
Please, Add link from Jenkins with screenshot

**Review by manowarqqq — COMMENTED** [review]
-

**Review by manowarqqq — COMMENTED** [review]
-

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-progress-bar.cy.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+let user;
+const invalidBarcode = getRandomPostfix();
+const userBarcodesFileName = `userBarcodes-val-inval${getRandomPostfix()}.csv`;
+
+describe('Bulk-Edit', () => {
+  describe('csv approach', () => {
+    before('create test data', () => {
+      cy.createTempUser([
+        permissions.bulkEditLogsView.gui,
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-progress-bar.cy.js:19]: Permissions.bulkEditLogsView.gui,
        Permissions.bulkEditUpdateRecords.gui,
        Permissions.uiUserEdit.gui,

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-progress-bar.cy.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    after('delete test data', () => {
+      cy.getAdminToken();
+      Users.deleteViaApi(user.userId);
+      FileManager.deleteFile(`cypress/fixtures/${userBarcodesFileName}`);
+    });
+
+    it(
+      'C353529 Verify progress bar for uploading identifier files(firebird) (TaaS)',
+      { tags: [testTypes.smoke, devTeams.firebird] },
```
Comment (manowarqqq) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-progress-bar.cy.js:44]: { tags: [TestTypes.smoke, DevTeams.firebird] },

**Review by manowarqqq — CHANGES_REQUESTED** [review]

**Review by nayimovag — APPROVED** [review]

**Review by manowarqqq — APPROVED** [review]

**Pull request merged**